libkbfs/block_disk_store.go

```diff
@@ -540,3 +540,7 @@ func (s *blockDiskStore) remove(id kbfsblock.ID) error {
 	}
 	return err
 }
+
+func (s blockDiskStore) clear() error {
```

Comment:
Looks like this is unused?  Did you mean to call it when clearing the block journal?